README.md

```diff
@@ -123,6 +123,18 @@ when a parent class is used to refer to a child class object. **Operator
 overloading** is one form of polymorphism. It is not used in Javascript but is a
 feature of C++, Scala, Ruby, Haskell and Rust.
 
+**Progressive Web App (PWA)** - PWAs are traditional websites that are enhanced
+with modern web technologies, allowing them to provide a more app-like
+experience. They offer functionality such as being saved as a tile on your
+mobile home screen, working offline and push notifications. The “progressive”
+part means they’re “progressively enhanced” with newer features, which means
+they’ll also work in older browsers that don’t support the new features. Unlike
+native apps, they don't have to be updated/listed in the app store although it
+is possible to list them in the app store when this is desired.
+
+**Regression Testing** - the re-running of existing tests to ensure that new
+changes/fixes have not broken any existing functionality.
```

Comment:
@Cleop you've done a lot more research into Regression Testing than this brief definition exposes.
Please consider linking to the issue where you go into a lot more detail, or add a second line with more detail here and link to the extended definition you found most useful. 👍